constructorio-client/src/test/java/io/constructor/client/ConstructorIOQuizResultsConfigTest.java

```diff
@@ -0,0 +1,98 @@
+package io.constructor.client;
+
+import static org.junit.Assert.*;
+
+import io.constructor.client.models.QuizResultsConfigResponse;
+import org.json.JSONObject;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ConstructorIOQuizResultsConfigTest {
+
+    private final String quizKey = System.getenv("TEST_REQUEST_API_KEY");
+    private final String quizId = "test-quiz";
+
+    @BeforeClass
+    public static void init() {}
+
+    @Rule public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void QuizResultsConfigShouldReturnResuls() throws Exception {
```

Comment:
Should we change the test name to `QuizResultsConfigShouldReturnAResponse` since it doesn't return results?

This would apply to the test names below as well: `Result` => `Response`